api/formatters/covjson.py

```diff
@@ -28,7 +29,15 @@
 
 
 def make_parameter(ts_mdata):
+    custom_fields = {
+        "rodeo:standard_name": ts_mdata.standard_name,
+        "rodeo:level": float(ts_mdata.level) if is_float(ts_mdata.level) else 0.0,
+        "rodeo:function": ts_mdata.function,
+        "rodeo:period": ts_mdata.period,
+    }
+
     return Parameter(
+        # TODO: Change description, as we know have the explicit fields?
```

Comment:
In my opinion for example: `air_pressure at 1.5m PT1M mean` is a good description of the parameter and could be kept as such.

Reply:
Fixed in: https://github.com/EURODEO/e-soh/pull/146/commits/19550f0927d6e09cca51a838c6375284245390c9